### ISSUE TITLE
Gauge Card is not working

### ISSUE THREAD

**Issue by Parchbrot** [marcokreeft87/room-card#61]
When I add a gauge into the card, sometimes it is shown till I reload the page or switch between the dashboard views

![gauge](https://user-images.githubusercontent.com/4101312/192166224-3d4d3825-c4ac-423f-92ba-f5fd15e0fdf7.png)

code:
```
type: custom:room-card
title: Solar
entity: Solar
icon: fas:solar-panel
show_icon: true
cards:
  - type: gauge
    entity: sensor.solartotalpower
    max: 300
    needle: false
    name: Aktuell
```

In other "default" cards the default gauge is working. Tested with other themes and no theme! Restarted multiple times. Redownloaded the frontend card, too.
Everytime same error: n.setConfig is not a function

**Comment by marcokreeft87** [comment]
Yes, this is a known issue. The problem lies with the gauge card. It doesn't have the setconfig method like the error says. I have to find how to deal with that in the room card.

I hope to find it soon, because I also have this problem with the custom thermostat card.

**Comment by marcokreeft87** [comment]
Should be fixed with v1.05.02
